terraform/modules/opensearch/examples/basic-configuration/main.tf

```diff
@@ -0,0 +1,11 @@
+module "opensearch" {
+  source = "../.."
+
+  domain_name                 = "program-tier-app-opensearch"
```

Comment:
same comment as the main file

Reply:
- refined to adopt resource_prefix standard.